logging/lib/src/logging_integration.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:async';
+
+import 'package:logging/logging.dart';
+import 'package:sentry/sentry.dart';
+import 'version.dart';
+import 'extension.dart';
+
+/// An [Integration] which listens to all messages of the
+/// [logging](https://pub.dev/packages/logging) package.
+class LoggingIntegration extends Integration<SentryOptions> {
+  /// Creates the [LoggingIntegration].
+  ///
+  /// All log events equal or higher than [minBreadcrumbLevel] are recorded as a
+  /// [Breadcrumb].
+  /// All log events equal or higher than [minEventLevel] are recorded as a
+  /// [SentryEvent].
+  LoggingIntegration({
+    Level minBreadcrumbLevel = Level.INFO,
+    Level minEventLevel = Level.SEVERE,
+  })  : _minBreadcrumbLevel = minBreadcrumbLevel,
+        _minEventLevel = minEventLevel;
+
+  final Level _minBreadcrumbLevel;
+  final Level _minEventLevel;
+  late StreamSubscription<LogRecord> _subscription;
+  late Hub _hub;
+
+  @override
+  FutureOr<void> call(Hub hub, SentryOptions options) {
+    _hub = hub;
+    _setSdkVersion(options);
```

Comment:
@ueman @marandaneto Should the SDK version be set here? This reports `sentry.dart.logging` .

Reply:
Does it also report `sentry.dart.logging` for exceptions captured by `sentry_flutter` or `sentry`? 
In general, every exception captured by `sentry_logging` should have `sentry.dart.logging` as its origin.

Reply:
From what I understand, this is called only once when the integration is registered and overrides the original SDK.

Reply:
Looks like this should not be set https://github.com/getsentry/sentry-dart/blob/main/logging/lib/src/logging_integration.dart#L49-L50
Since its an Integration rather than a top-level package, adding to the package list https://github.com/getsentry/sentry-dart/blob/main/logging/lib/src/logging_integration.dart#L54
Adding to the integration list https://github.com/getsentry/sentry-dart/blob/main/logging/lib/src/logging_integration.dart#L38 and setting the logger https://github.com/getsentry/sentry-dart/blob/main/logging/lib/src/extension.dart#L27 should have been enough.
Would you like to submit a PR @kuhnroyal ?